Source/WebGPU/WebGPU/Texture.mm

```diff
@@ -2584,10 +2585,10 @@ static WGPUExtent3D imageCopyTextureSubresourceSize(const WGPUImageCopyTexture&
 bool Texture::isValidImageCopyDestination(WGPUTextureFormat format, WGPUTextureAspect aspect)
```

Comment:
If this only deals with depth/stencil formats, I feel like this function has the wrong name; Image feels to generic.

Reply:
I agree, maybe `isValidDepthStencilCopyDestination` and `isValidDepthStencilCopySource`?